SUMMARY.md

```diff
@@ -22,6 +22,7 @@
 * [Connecting UI & Logic](core-concepts/connecting-ui-and-logic/README.md)
   * [Working with JS libraries](core-concepts/connecting-ui-and-logic/working-with-js-libraries.md)
   * [Internal Functions](core-concepts/connecting-ui-and-logic/internal-functions.md)
+  * [Data Sharing Across Pages](core-concepts/connecting-ui-and-logic/sharing-data-across-pages.md)
```

Comment:
Can we change the text to 
Sharing Data Across Pages

Reply:
Done